discord/webhook.py

```diff
@@ -642,6 +652,62 @@ def edit(self, **kwargs):
 
         return self._adapter.edit_webhook(**payload)
 
+    async def _async_move(self, channel_id, token, payload):
+        await self._adapter.move_webhook(token, **payload)
+        self.channel_id = channel_id
+
+    def _sync_move(self, channel_id, token, payload):
+        self._adapter.move_webhook(token, **payload)
+        self.channel_id = channel_id
+
+    def move_to(self, channel):
+        """|maybecoro|
+
+        Moves this Webhook.
+
+        If the webhook is constructed with a :class:`RequestsWebhookAdapter` then this is
```

Comment:
If you get Webhook from Client/TextChannel/Guild, it will always use Async adapter, so this should be omitted.